DOCS.md

```diff
@@ -1,3 +1,14 @@
+# --- WARNING ---
+## Pelias v1.0 will be released in September 2015!!!
+## All users must obtain FREE developer keys in order to
+## continue using the Pelias service after v1.0 has been released.
+## There will be breaking changes to the API,
+## so action must be taken to upgrade client code.
+## Backwards compatibility will be maintained through November 2015,
+## after which the old service will be deprecated and taken off line.
+# ----- END -----
+
+
```

Comment:
maybe, we should figure out the PR strategy overall and list all the places deprecation/release notices are needed @riordan